src/components/item-details/bid-tab/history-tab-content.tsx

```diff
@@ -1,23 +1,62 @@
-import TopSeller from '@components/top-seller/top-seller-2';
-import { History } from '@types';
+import { History, Item } from '@types';
+import Anchor from '@ui/anchor';
+import { getTimeAgo } from '@utils/getTimeAgo';
+import clsx from 'clsx';
+import Image from 'next/image';
+import useSWR from 'swr';
+import { itemService } from '../../../features/leda-nft/services/item.service';
 
 type Props = {
-  history: History[];
+  item: Item;
 };
+const fetchHistory = async (itemId: string) =>
+  itemService.findHistoryByItemId(itemId).then((data) => data.reverse());
 
-const HistoryTabContent = ({ history }: Props) => (
-  <div>
-    {history?.map((item: History) => (
-      <TopSeller
-        key={item.id}
-        name={item.user.name}
-        eth={item.amount}
-        path={item.user.slug}
-        time={item.bidAt}
-        image={item.user.image}
-      />
-    ))}
-  </div>
-);
+export const HistoryTabContent = ({ item }: Props) => {
+  const { data: history } = useSWR<History[]>(item.itemId, fetchHistory);
 
-export default HistoryTabContent;
+  if (!history?.length) {
+    return (
+      <div className="text-center my-5">
+        <h3>No history found</h3>
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      {history?.map((e) => (
+        <div className="top-seller-inner-one" key={e.id}>
+          <div className="top-seller-wrapper">
+            {item.image?.url && (
+              // We need a image profile
```

Comment:
I would rather not to render a hardcoded image. We could try with the nft image though. Otherwise drop the image placeholder since this will go to prod soon.

Reply:
I don't wanted to add the image of item because it's redundant